### PR TITLE
HDDS-3318. Support /close command in the Github comments

### PR DESCRIPTION
**Pull request by elek**
## What changes were proposed in this pull request?

As it's discussed during the last community meeting we can close pending pull requests after 30 days. /close command can make it easier.

We can also add pending label to the pending pull requests.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3318

## How was this patch tested?

On my personal fork. But I will try to test it on this pr.

### CODE REVIEW

**Comment by elek** [comment]
> But I will try to test it on this pr.

It's not working. But it's tested on this PR:

https://github.com/elek/hadoop-ozone/pull/10

File: .github/comment-commands/close.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#doc: Close pending pull request temporary
+# shellcheck disable=SC2124
+MESSAGE="Thank you very much the patch. I am closing this issue __temporary__ as there was no 
+activity recently and it's pending on the author of the PR.
```
Comment (adoroszlai) [inline .github/comment-commands/close.sh:20]:
```suggestion
MESSAGE="Thank you very much for the patch. I am closing this PR __temporarily__ as there was no 
activity recently and it is waiting for response from its author.
```

File: .github/comment-commands/close.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+It doesn't mean that this PR is not important or ignored: feel free to reopen the PR at any time.
+
+It means only that attention of commiters is not required. To get faster review for all the PRs, 
+we prefer to clean the queue to show which PRs require review or feedback.
```
Comment (adoroszlai) [inline .github/comment-commands/close.sh:25]:
```suggestion
It only means that attention of committers is not required. We prefer to keep the review queue clean. This ensures PRs in need of review are more visible, which results in faster feedback for all PRs.
```

File: .github/comment-commands/close.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+If you need ANY help to finish this PR, please contact with the community on the mailing list or the slack channel.
+
+(See the README about the details of chat/mailing list: https://github.com/apache/hadoop-ozone)"
```
Comment (adoroszlai) [inline .github/comment-commands/close.sh:29]:
```suggestion
If you need ANY help to finish this PR, please [contact the community](https://github.com/apache/hadoop-ozone#contact) on the mailing list or the slack channel.
```

File: .github/comment-commands/close.sh (hunk 2 above)
Comment (elek) [reply .github/comment-commands/close.sh:25]: typo fixes 2

**Comment by elek** [comment]
Thanks the review @adoroszlai (fixing my language mistakes are always a big help for me), I am merging it to the trunk.

**Pull request closed without merging**